hack/update-codegen.sh

```diff
@@ -39,13 +39,30 @@ kube::codegen::gen_helpers \
   "${TRAINER_ROOT}/pkg/apis"
 
 # Generate clients.
+externals=(
+  "sigs.k8s.io/jobset/api/jobset/v1alpha2.JobSetSpec:sigs.k8s.io/jobset/client-go/applyconfiguration/jobset/v1alpha2"
```

Comment:
Do we need to add `batch/v1 Job` as well? Or this JobSet import can automatically handle the batch/v1 Job typed?

Reply:
You're absolutely right!

I've created https://github.com/kubernetes-sigs/jobset/pull/782 so it'll be pulled transitively.

Reply:
Great effort across projects!
/lgtm
/approve